modal-js/src/sandbox.ts

```diff
@@ -36,6 +37,22 @@ export type StdioBehavior = "pipe" | "ignore";
  */
 export type StreamMode = "text" | "binary";
 
+/** Options for `Sandbox.setTags()`. */
+export type SandboxSetTagsOptions = {
+  /** Override environment for the request; defaults to current profile. */
+  environment?: string;
```

Comment:
What would overriding the environment mean for setting tags? The sandbox must already exist in an environment right?

Reply:
I pinged @danielshaar about this in a dm, since this goes back to the data model in the backend sandbox tag table, but for now I think we should remove this option from SandboxSetTagsOptions (the Python SDK also doesn't expose this but falls back on the current context environment (also not great but probably right in most cases)

Reply:
Nice catch! I made this to bring behavior in line with Python for now, until we change it server-side: https://github.com/modal-labs/libmodal/pull/95